client/server/isomorphic-routing/index.js

```diff
@@ -12,6 +12,7 @@ export function serverRouter( expressApp, setUpRoute, section ) {
 		expressApp.get(
 			route,
 			( req, res, next ) => {
+				req.context.usedSSSRHandler = true;
```

Comment:
The idea is to track when we're using the SSR pipeline to resolve a request, not necessarily that SSR was used to generate the markup. Perhaps it'd be worth logging both?